developer/docs/arvo/http.md

```diff
@@ -145,63 +162,76 @@ progress, then we send an HTTP request.
+> construct our list of moves using the common square bracket syntax (as shown 
+> below this note), but our move is too wide to use wide form. Instead,
+> we use a combination of cell-construction runes for what we need to produce. 
+> The first rune (`:~`) constructs a null-terminated list. The second (`:*`) 
+> constructs an n-tuple. The final product is a null-terminated list with one 
+> element (itself a list) which contains our move.
 
+Check out how we could produce the same list of moves using wide form: 
 ```
-:*  bone=ost.hid                                        :: the move
-    term=%hiss
-    wire=/request
-    unit-iden=~
-    mark=%httr
-    cage=[%purl (need (epur target))]
-==
+[ost.bow %hiss /request ~ %httr %purl (need (de-purl:html target))]~
 ```
 
-When the HTTP response comes back, we handle it with
-`++sigh-httr`, which, along with the wire the request was sent
-on, takes the status code, the response headers, and the response
-body.
+> With this, notice how the move is followed by a `~`. This is a convenient 
+> shortcut for creating a list of a single element. It's part of a small 
+> family of creating a list of a single element. It's part of a small family 
+> of such shortcuts. `~[a b c]` is `[a b c ~]`, `[a b c]~` is `[[a b c] ~]` 
+> and `\`[a bc]` is `[~ a b c]`. These may be mixed and matched to create 
+> various convoluted structures and emojis. The problem with this code is that 
+> it extends wider than 55 columns, which is beyond what is recommended.
+
+When the HTTP response comes back, we handle it with `++sigh-httr`, which, 
+along with the wire the request was sent on, takes the status code, the 
+response headers, and the response body.
 
-We check whether the status code is between 200 and 300.  If so,
-all is well, and we print a message saying so.  We start the
-timer for ten seconds after the present time.
+We check whether the status code is between 200 and 300. If so, all is well, 
+and we print a message saying so. We start the timer for ten seconds after the 
+present time.
 
-If we got a bad status code, then we print out the entire
-response and start the timer again.
+If we got a bad status code, then we print out the entire response and start 
+the timer again.
 
-After ten seconds, arvo will give us a `%wake` event, which will
-be handled in `++wake-timer`.  If we're still supposed to keep
-monitoring, we send the same HTTP request as before.  Otherwise,
-we set `in-progress` to false.
+After ten seconds, arvo will give us a `%wake` event, which will be handled in 
+`++wake-timer`. If we're still supposed to keep monitoring, we send the same 
+HTTP request as before. Otherwise, we set `in-progress` to false.
 
 Let's try it out:
 
 ```
-~fintud-macrep:dojo> |start %up
+~fintud-macrep:dojo/examples> |start %up
 >=
-~fintud-macrep:dojo> :examples-up &atom 'http://www.google.com'
+~fintud-macrep:dojo/examples> :up &atom 'on'
```

Comment:
It's probably better style to pass in `%on` instead of `'on'`, though technically it doesn't matter.